_data/latest.yml

```diff
@@ -1,9 +1,6 @@
 ---
-title: DevRel Salon № 8
-when: Wednesday, 30 May, 18h30
-where: Chez Josephine, 6 place du marché Sainte-Catherine
-location: https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d1312.5759443217125!2d2.3618636856639315!3d48.85531397307435!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x47e671fe240767d7%3A0x1d910c034bd1b629!2s6+Place+du+March%C3%A9+Sainte-Catherine%2C+75004+Paris-4E-Arrondissement!5e0!3m2!1sen!2sfr!4v1527239348533
-content: <p>Join us for Printemps et Peut Être Pétanque à Paris. Enjoy some quiet terrace space in the sweet weather of Paris's spring, social time with colleagues, and maybe we'll even throw heavy objects competitively. Of course there will be plenty of shop talk and networking with cool members of the DevRel community. Spots are limited, so be sure to <a href="https://www.meetup.com/Paris-DevRel-Salon/events/250873651/">RSVP</a>!</p>
-  <p>Please <a href="https://www.meetup.com/Paris-DevRel-Salon/events/250873651/">RSVP</a>. We'll be there waiting for you.</p>
-
-
+title: "DevRel Salon № 9"
+when: "Monday, 3 December, 2018"
+where: "Datadog, 1 rue de Metz, 75010"
+location: "https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d2624.348275091946!2d2.35226171560162!3d48.87063690779842!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x47e66e11837fb59b%3A0x2d53d091389aba01!2sDatadog!5e0!3m2!1sen!2sus!4v1543413250168"
+content: "<p>For the December edition of the Paris DevRel Salon, we welcome the Salon’s founder, Don Goodman-Wilson (@DEGoodmanWilson) back for an evening of discussion and exchange!</p><p>Our topic: Ethics in Developer Relations<br />Our host: Datadog (1 rue de Metz, 75010 Paris), who are supplying nibblies and drinks</p><p>We don’t want any food to go to waste, so RSVP is required. Thanks!</p><p>And now for more good news: the DevRel Salon is coming back in a big way for 2019! Highlights include:<br /><ul><li>A new rhythm, with an event the first week of every month.</li><li>A new cycle, with a traditional meet-up one month, and an informal get-together the next.</li><li>Better scheduling, because we’re all busy, so dates and locations will be announced months ahead of time.</li><li>More collaboration! Send your topics and venue suggestions to us on Twitter (@devrelsalon) or email (em@ail).</li></ul></p><p>We’re still hammering out the details, but our upcoming events in 2019 are slated for 7 January, 4 February, and 4 March. Please send us your suggestions for venues - both formal and informal - and discussion topics.</p><p>Can’t wait to see you in December, and all through 2019! It’s gonna be grand! 🎉</p>"
```

Comment:
```suggestion
content: "<p>For the December edition of the Paris DevRel Salon, we welcome the Salon’s founder, Don Goodman-Wilson (@DEGoodmanWilson) back for an evening of discussion and exchange!</p><p>Our topic: Ethics in Developer Relations<br />Our host: Datadog (1 rue de Metz, 75010 Paris), who are supplying nibblies and drinks</p><p>We don’t want any food to go to waste, so RSVP is required. Thanks!</p><p>And now for more good news: the DevRel Salon is coming back in a big way for 2019! Highlights include:<br /><ul><li>A new rhythm, with an event the first week of every month.</li><li>A new cycle, with a traditional meet-up one month, and an informal get-together the next.</li><li>Better scheduling, because we’re all busy, so dates and locations will be announced months ahead of time.</li><li>More collaboration! Send your topics and venue suggestions to us on Twitter (@devrelsalon) or email (howdy@devrel.salon).</li></ul></p><p>We’re still hammering out the details, but our upcoming events in 2019 are slated for 7 January, 4 February, and 4 March. Please send us your suggestions for venues - both formal and informal - and discussion topics.</p><p>Can’t wait to see you in December, and all through 2019! It’s gonna be grand! 🎉</p>"
```

Reply:
or `paris@devrel.salon`